router/Dockerfile

```diff
@@ -1,29 +1,7 @@
-FROM alpine:3.2
-
-# install common packages
-RUN apk add --update-cache \
-	bash \
-	curl \
-	geoip \
-	libssl1.0 \
-	openssl \
-	pcre \
-	sudo \
-	&& rm -rf /var/cache/apk/*
-
-# install confd
-RUN curl -sSL -o /usr/local/bin/confd https://s3-us-west-2.amazonaws.com/opdemand/confd-git-73f7489 \
-  && chmod +x /usr/local/bin/confd
-
-# add nginx user
-RUN addgroup -S nginx && \
-  adduser -S -G nginx -H -h /opt/nginx -s /sbin/nologin -D nginx
+FROM quay.io/hearst/deis-router:v.1.12.2-21-af7012a
 
 COPY rootfs /
 
```

Comment:
Do we want to recopy all of rootfs here or just our changed file?

Reply:
Is this a cover all copy incase any other files in the future are to be altered, or really only the nginx template that would get changed?

Reply:
I think it's better to only copy the specifics from this ticket, there's no guarantee the same approach would work for other changes, so i'd rather modify the dockerfile accordingly if that change is needed.